client/src/components/SearchBar/SearchBar.jsx

```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
 const SearchBar = function() {
+  const [ searched, setSearched ] = useState('');
 
+  useEffect(() => {
+    if (searched.length >= 3) {
+      axios.get('http://52.26.193.201:3000/qa/5')
+        .then(res => console.log(res.data))
+        .catch(err => console.log(err));
+    }
+  }, [searched]);
+
+  const handleChange = function(event) {
+    const { value } = event.target;
+    setSearched(value);
+  };
+
+  return (
+    <div className='question-searchbar'>
+      <form>
+      <label>QUESTIONS & ANSWERS</label>
+      <input
+        placeholder='HAVE A QUESTION? SEARCH FOR ANSWERS...'
+        aria-label='searchbar'
+        value={searched}
```

Comment:
Does your search handle text for searching products or does it handle numbers for searching product numbers. It looks like right now that you have the get request hardcoded for a certain product, which works, but I was not sure what input you were looking to get with a user

Reply:
It should just be handling text the user types in and this using the text to perform a key word search in the course of an API request. For now, I've only focused on getting the bar to take in the text and save it in state.